strongbox-client/src/test/java/org/carlspring/strongbox/client/ArtifactResolverIntegrationTest.java

```diff
@@ -86,9 +85,16 @@ public MultiHttpClientConnThread(RestArtifactResolver artifactResolver,
         @Override
         public final void run()
         {
-            CloseableRestResponse response = artifactResolver.get(url);
-            ResourceCloser.close(response, LOGGER);
+            try (CloseableRestResponse response = artifactResolver.get(url))
+            {
+                // Left empty because the original method had no other logic than the now-removed
+                // closure of a CloseableRestResponse
+            }
+            catch (Exception e)
+            {
+                LOGGER.error(e.getMessage(), e);
+            }
         }
 
     }
-}
+}
```

Comment:
All source and resource files should have an empty line at the end, to guarantee that all editors and build tools can read them properly.